spring-security-kerberos-core/src/main/java/org/springframework/security/kerberos/authentication/KerberosTicketValidation.java

```diff
@@ -16,12 +16,14 @@
 	private final byte[] responseToken;
 	private final GSSContext gssContext;
 	private final String servicePrincipal;
+	private final GSSCredential delegationCredential;
 
-	public KerberosTicketValidation(String username, String servicePrincipal, byte[] responseToken, GSSContext gssContext) {
+	public KerberosTicketValidation(String username, String servicePrincipal, byte[] responseToken, GSSContext gssContext, GSSCredential delegationCredential) {
```

Comment:
Rather than removing the constructor we should add an additional constructor to remain passive. Can you update the PR?